doozerlib/distgit.py

```diff
@@ -515,6 +515,23 @@ def _generate_osbs_image_config(self):
         if arches:
             config_overrides.setdefault('platforms', {})['only'] = arches
 
+        # Request OSBS to apply specified tags to the newly-built image as floating tags.
+        # See https://osbs.readthedocs.io/en/latest/users.html?highlight=tags#image-tags
+        #
+        # Include the UUID in the tags. This will allow other images being rebased
+        # to have a known tag to refer to this image if they depend on it - even
+        # before it is built.
+        floating_tags = {"latest", f"{version}.{self.runtime.uuid}"}
```

Comment:
we did not have `latest` before and while i don't _think_ it will cause conflict in the prod registry, it doesn't seem like a good idea to set it (especially since i keep trying to discourage people from using it at all).

Reply:
I added `latest` tag just for keeping compatibility. Now the latest tag is removed.